apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/domain-step/index.tsx

```diff
@@ -12,26 +12,15 @@ import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
 import DomainPickerFSE from '../../../../editor-domain-picker/src/domain-picker-fse';
 import './styles.scss';
 
-const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onNextStep } ) => {
-	const handleBack = () => {
-		onPrevStep?.();
-	};
-
+const DomainStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep } ) => {
 	return (
 		<LaunchStep className="nux-launch-domain-step">
 			<div className="nux-launch-step__header">
-				<div className="nux-launch-step__heading">
-					<h1 className="nux-launch-step__title">
-						{ __( 'Choose a domain', 'full-site-editing' ) }
-					</h1>
-					<p className="nux-launch-step__subtitle">
+				<div>
+					<Title>{ __( 'Choose a domain', 'full-site-editing' ) }</Title>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.